### PR TITLE
Mobile-First Pull Request

### PR DESCRIPTION
**Pull request by skabeo**
Added an index.html file containing the content of the project, a styles folder containing the styles for the HTML file. Lastly, added an images folder containing every image used on the project.

### CODE REVIEW

File: styles/general.css [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+}
+
+body {
+  background-color:  #00040F;
+  font-family: 'Poppins', sans-serif;
+  color: white;
+  margin: 0;
+
+  padding: 32px 24px;
+}
```
Comment (hiromataba) [inline styles/general.css]: - [ ] For the background color please make sure that it matches the original design 

- Your design 👇 

image

- Origina design 👇 

image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+  <link rel="stylesheet" href="styles/general.css">
+  <link rel="stylesheet" href="styles/nav.css">
+  <link rel="stylesheet" href="styles/info.css">
+  <link rel="stylesheet" href="styles/footer.css">
+</head>
+<body>
+  <main>
+    <header id="header">
+      <div class="header-container">
+        <p class="text-gradient">Welcome</p>
```
Comment (hiromataba) [inline index.html]: - [ ] [ OPTIONAL] I think it would be better to use you name here instead of `Welcome`

File: index.html [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+        <img class="menu" src="img/menu.svg" alt="menu">
```
Comment (hiromataba) [inline index.html]: - [ ] Please use the same hamburger as the original design and you should use a the `nav` tag for the menu hamburger

- Original design 👇 
image

- Your design 👇 

image

File: index.html (hunk 2 above)
Comment (hiromataba) [inline index.html]: - [ ] For the logo you should use the `a` tag so that it maybe clickable because a logo is always a link

File: index.html [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+        </div>
+        <div class="img-contaner">
+          <a href="https://twitter.com/black_okyere"><img src="img/twitter.svg" ></a>
+        </div>
+        <div class="img-contaner">
+          <a 
+            href="https://www.linkedin.com/in/okyere-spencer-9b602623b/">
+            <img src="img/linkedin.svg" ></a>
+        </div>
+      </div>
```
Comment (hiromataba) [inline index.html]: - [ ] You should use the `ul` tag here as you did for the other icons to list the icons and `li` per icon

File: styles/footer.css [hunk 5]
```diff
@@ -0,0 +1,49 @@
+.footer {
+  display: flex;
+  justify-content: center;
+  align-items: center;
```
Comment (hiromataba) [inline styles/footer.css:4]: - [ ] You did great using display flex 😉

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi @skabeo     :wave: :wave: 

almost-there

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Your PR has a proper title and description  :+1: 
- Your linters are passing :heavy_check_mark:  
- Profesional README file :rocket: 

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+        </div>
+        <div class="items">
+          <h4 class="text-gradient">Frameworks</h4>
+          <ul>
+            <li>Bootstrap</li>
+            <li>Ruby on Rails</li>
+            <li>RSPec</li>
+            <li>CapyBara</li>
+            <li>Solenium</li>
+          </ul>
```
Comment (aamirkhan2478) [inline index.html:67]: - [ ] Well done you did a great job :+1:  but still some little bit changes are required please check below:

Your Design:
screencapture-127-0-0-1-5500-2023-02-16-23_30_06

Figma Design:
219378099-b12fb80d-4e20-4b79-ba3b-034a88fb9bd8

- [ ] Please add this blur circle here then your design will be the same as the Figma design.

File: styles/general.css [hunk 7]
```diff
@@ -0,0 +1,75 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500&display=swap');
+
+*::before,
+*::after {
+  margin: 0;
+}
+
+body {
+  background-color:  #191765;
+  font-family: 'Poppins', sans-serif;
```
Comment (aamirkhan2478) [inline styles/general.css:10]: - [ ] Kindly use the same font family as the Figma font family which is `Roboto`.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+    <hr class="line">
+
+    <section id="social-links">
+      <div class="social-links">
+        <ul class="img-container">
+          <li><a href="https://www.instagram.com/___spec/"><img src="img/instagram.svg" ></a></li>
+          <li><a href="https://twitter.com/black_okyere"><img src="img/twitter.svg" ></a></li>
+          <li><a 
+            href="https://www.linkedin.com/in/okyere-spencer-9b602623b/">
+            <img src="img/linkedin.svg" ></a></li>
```
Comment (aamirkhan2478) [inline index.html]: - [ ] Kindly add all social media apps as mentioned in the Figma design:

Screenshot from 2023-02-16 23-57-49

**Review by aamirkhan2478 — CHANGES_REQUESTED** [review]
Hi @skabeo , 



Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

 ## Highlights
✔️ Good Effort
✔️ Your design is good.
✔️ Your PR has a proper title and description
✔️ Your linters are passing 
✔️ Profesional README file

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 9]
```diff
@@ -0,0 +1,100 @@
...
+  <link rel="stylesheet" href="styles/general.css">
+  <link rel="stylesheet" href="styles/nav.css">
+  <link rel="stylesheet" href="styles/info.css">
+  <link rel="stylesheet" href="styles/footer.css">
+</head>
+<body>
+  <main>
+    <header id="header">
+      <div class="header-container">
+        <a href="https://twitter.com/black_okyere"><p>Skabeo</p></a>
```
Comment (wayungi) [inline index.html:18]: - [OOTIONAL] The logo should be a link that links back to the home page. You link leads to twitter 🐦

**Review by wayungi — APPROVED** [review]
Hi @skabeo , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

image

## Highlights
- [ ] Profesional README ✔️ 
- [ ] All linter checks passing ✔️ 
- [ ] Good commit messages ✔️ 
- [ ] No merge conflicts ✔️ 
- [ ]  Correct GitHub flow ✔️ 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**